storage/objects/objects_test.go

```diff
@@ -360,14 +360,14 @@ func TestV4SignedURL(t *testing.T) {
 
 	bucketName := tc.ProjectID + "-signed-url-bucket-name"
 	objectName := "foo.txt"
-	serviceAccount := os.Getenv("GOOGLE_APPLICATION_CREDENTIALS")
-	if serviceAccount == "" {
+
+	if os.Getenv("GOOGLE_APPLICATION_CREDENTIALS") == "" {
```

Comment:
I think we can assume that users need to have credentials in order to run these tests.

Reply:
You're right, I'm deleting the check. It will error with an appropriate message if the credentials are not set.